docs/Changelog.md

```diff
@@ -790,6 +790,58 @@ This version of the operator has been available since version 1 of the default O
 <dd>Constrain input and output types to float tensors.</dd>
 </dl>
 
+### <a name="Conv_Integer-1"></a>**Conv_Integer-1**</a>
+
+  The integer convolution operator consumes an input tensor, a filter and a padding value, and computes the output. The production MUST never overflow. The accumulation may overflow if and only if in 32 bits.
+
+#### Version
+
+This version of the operator has been available since version 1 of the default ONNX operator set.
+
+#### Attributes
+
+<dl>
+<dt><tt>auto_pad</tt> : string</dt>
```

Comment:
I'm not sure we need to support it for quantized convolution

Reply:
Hmm... I saw "DEPRECATION NOTE: auto_pad is only intended to support legacy uses, and for framework authors, one is explicitly encouraged to use explicit padding specified in the pads attribute." in Conv.

If we drop it in Conv, then we should also drop it here.
Two should be consistent.